manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-5-0-rc-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ScaryRawr.SL
+PackageVersion: 5.06.0
+PackageLocale: en-US
+Publisher: ScaryRawr
```

```suggestion
Publisher: sl
```
This needs to match Control Panel, or `AppsAndFeaturesEntries` need to be specified in order for WinGet to properly detect the package